test/run-helm-tests.sh

```diff
@@ -0,0 +1,44 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+# This runs helm tests for a release using `helm test` command
+if [[ -n "${KIND_E2E}" ]]; then
+    K8S_VERSION=${KUBERNETES_VERSION:-v1.18.2}
+    curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+    wget https://github.com/kubernetes-sigs/kind/releases/download/v0.9.0/kind-linux-amd64
+    chmod +x kind-linux-amd64
+    mv kind-linux-amd64 kind
+    export PATH=$PATH:$PWD
+    kind create cluster --image kindest/node:"${K8S_VERSION}" --config=./hack/kind_config.yaml
+    export KUBECONFIG="$(kind get kubecofnig-path)"
+    docker pull kubernetes/pause
+    kind load docker-image kubernetes/pause
+    kind get kubeconfig > /tmp/admin.conf
+    export KUBECONFIG="/tmp/admin.conf"
+    mkdir -p ~/gopath/src/sigs.k8s.io/
+    mv ~/gopath/src/github.com/kubernetes-sigs/descheduler ~/gopath/src/sigs.k8s.io/.
```

Comment:
```suggestion
```

Reply:
@seanmalloy shall I keep this line: ```export KUBECONFIG="$(kind get kubecofnig-path)"``` or remove that one too? I'm guessing since we aren't running `kubectl` commands so that might not be required right?

Reply:
Updated the code by removing mentioned lines.